dynamodb_revisions/other_id_ref.py

```diff
@@ -0,0 +1,88 @@
+"""Other_id ref, update normalized data for production DynamoDB instance."""
+from therapy.database import Database
+from timeit import default_timer as timer
+from boto3.dynamodb.conditions import Attr
+from therapy.schemas import SourceName
+
+db = Database()
+db.batch = db.therapies.batch_writer(overwrite_by_pkeys=['label_and_type',
+                                                         'concept_id'])
+
+normalized_srcs = {SourceName.WIKIDATA.value,
+                   SourceName.RXNORM.value,
+                   SourceName.NCIT.value,
+                   SourceName.CHEMIDPLUS.value}
```

Comment:
I don't think this is a huge deal since (I think) this revision is only going to be used once, but we could have used one of the constants from `therapy/__init__.py`

Reply:
Yeah, I thought about this -- unfortunately we don't have a permitted sources variable with proper casing (ACCEPTED_SOURCES is all lower-case) but I guess we can just lowercase each `src_name` attribute